### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GiantLuigi4**

### CODE REVIEW

**Comment by GiantLuigi4** [comment]
I have no idea why, but this one random "a" bothers me

**Comment by Noaaan** [comment]
I do like that "completely" is also mispelled in this README

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Fabric mod where anyone can PR anything, concerning or not (no NSFW content). I'
 ## How can I ~~fuck up~~ contribute to this project ?
 
 1) Create what you want : you can do ANYTHING , adding blocks, items, whatever you want, mess with mixins, break vanilla mecanics, add dependencies, PR entire projects, as soon as it doesn't break other contributions and that it doesn't include any kind of virus, keylogger or backdoor. Game hacks are fine, system hacks are not.
-2) If possible, make it configurable. Not everyone wants to have a completly messed up rendering or a "concern" block that generates everywhere
```
Comment (TehcJS) [inline README.md:9]: this is a very controversial change. i don't believe it'll make it past reviewing phase

**Review by TehcJS — CHANGES_REQUESTED** [review]
delete this

File: README.md (hunk 1 above)
Comment (anatom3000) [reply README.md:9]: no, bc people generally add features that tend to break things/the entire game, so its good to remind people to make things optiionals

**Pull request closed without merging**